### PR TITLE
 Scale Kubernetes workloads based on events.md

### PR DESCRIPTION
**Pull request by yelghali**
@dubrie

### CODE REVIEW

File: docs/catalog/cloud/Scale Kubernetes workloads based on events.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- serverless
```
Comment (dubrie) [inline docs/catalog/cloud/Scale Kubernetes workloads based on events.md:7]: please add "cloud" to the list of tags

File: docs/catalog/cloud/Scale Kubernetes workloads based on events.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Scale Kubernetes workloads based on events
+
+## Description
+By default, Kubernetes scales workloads based on CPU and RAM utilization, in practice however, it is difficult to define the right thresholds for scaling based on CPU and RAM.
+
+Scaling your workload based on relevant business metrics such as HTTP requests, queue length, and cloud events can help reduce its resource utilization, hence its carbon emissions.
+
+## Solution
+
+Use [KEDA](https://keda.sh) when building event-driven kubernetes workloads, to allow scaling down to zero when there is no demand.
```
Comment (dubrie) [inline docs/catalog/cloud/Scale Kubernetes workloads based on events.md:23]: Is there a more generic approach that doesn't specifically require a vendor-based solution?  You can use KEDA as an example, but the solution should be vendor-agnostic.

Comment (dubrie) [inline docs/catalog/cloud/Scale Kubernetes workloads based on events.md:23]: I'm still not clear what the proposed solution is here. Is it to use metrics other than CPU and RAM utilization to scale your services? How is using 1000 HTTP requests or queue length > 100 more emissions efficient than CPU utilization?

File: docs/catalog/cloud/Scale Kubernetes workloads based on events.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, scaling based on events will impact:
+
+- `M`: By scaling the workloads based on demand and reducing resource utilization, we reduce the total embodied carbon emissions by the kubernetes cluster.
+
+## Assumptions
+
+In addition to scaling kubernetes workloads based on events, cluster auto-scaling and bursting capabilities are used to scale infrastructure based on events as well.
```
Comment (dubrie) [inline docs/catalog/cloud/Scale Kubernetes workloads based on events.md:35]: This feels like it would be better in the solution section, I'm not sure what assumption is trying to be referenced here.

**Review by dubrie — CHANGES_REQUESTED** [review]
@yelghali changes requested from initial review

File: docs/catalog/cloud/Scale Kubernetes workloads based on events.md (hunk 2 above)
Comment (yelghali) [reply docs/catalog/cloud/Scale Kubernetes workloads based on events.md:23]: Keda is OSS and works with any Kubernetes, i'll rephrase it to explicit it is only an enabler.

**Comment by dubrie** [comment]
Changes make this more clear. Thank you!

**Review by dubrie — APPROVED** [review]
